web/client/components/TOC/__tests__/Toolbar-test.jsx

```diff
@@ -530,7 +530,8 @@ describe('TOC Toolbar', () => {
                         minx: -10,
                         miny: -9
                     }, crs: 'EPSG:3003'
-                }
+                },
+                search: "fakeURL"
```

Comment:
The PR looks good, I would only suggest to use an object for the search.
an example:
```js
search: {
  type: "wfs",
  url: "/geoserver/wfs"
}
```